guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -26,11 +26,15 @@ You must assign every host that is managed by {Project} to a domain. Using the d
 Subnet::
 You must assign every host managed by {Project} to a subnet. Using subnets, {Project} can then manage IPv4 reservations. If there are no reservation integrations, you still must create and associate at least one subnet. When you manage a subnet in {Project}, you cannot create DHCP records for that subnet outside of {Project}. In {Project}, you can use IP Address Management (IPAM) to manage IP addresses with one of the following options:
 +
-* *DHCP*: the DHCP {SmartProxy} manages the assignment of IP addresses by finding the next available IP address starting from the first. Before assigning an IP address, {SmartProxy} sends an ICMP and TCP ping to verify whether the IP address is in use.
-* *Internal DB*: {Project} manages a list of available IP addresses in the PostgreSQL database. {Project} assigns the IP addresses automatically.
-* *Random DB*: {Project} manages a list of available IP addresses in the PostgreSQL database, but randomizes the results to avoid IPv4 race conditions.
+* *DHCP*: the DHCP {SmartProxy} manages the assignment of IP addresses by finding the next available IP address starting from the first address of the range skipping all addresses that are reserved. Before assigning an IP address, {SmartProxy} sends an ICMP and TCP ping to perform a brief check whether the IP address is in use. This check will not work for hosts which are turned off, so the DHCP option can be only used for subnets which are in full {Project} control without any hosts created externally. {SmartProxy} DHCP module retains offered IP addresses for short period of time to prevent collisions during concurrent access, therefore temporary "holes" in the IP range can exist.
```

Comment:
Foreman tries to "knock the doors" using PING command/protocol if the IP address is alive or not. The problem is, when a host with this IP assigned is literally powered off (turned off) or even (I missed this originally in my text) if it has a firewall configured in a way it would drop those packets, Foreman can make a false assumption the IP address is available.

Therefore users should be cautious and only use this for subnets which are under full control of Foreman.

Reply:
And what are those subnets that are in full Foreman control? Does it mean that the subnet must be managed by Foreman only, on not by any other soft?

Reply:
Yes not by other software or person (manually modifying configuration or database files).